event-statistics/src/main/resources/application.properties

```diff
@@ -9,7 +9,7 @@ mp.messaging.incoming.fights.connector=smallrye-kafka
 mp.messaging.incoming.fights.topic=fights
 mp.messaging.incoming.fights.auto.offset.reset=earliest
 mp.messaging.incoming.fights.broadcast=true
-mp.messaging.incoming.fights.value.deserializer=io.quarkus.sample.superheroes.statistics.listener.FightDeserializer
+mp.messaging.incoming.fights.enable.auto.commit=false
```

Comment:
That's the default.

Reply:
I took this right from https://quarkus.io/guides/kafka-schema-registry-avro#the-movie-consumer. The code snippet there has this. I wasn't sure if it was just showcasing the default, but the comment on the snippet (to me) made it seem like I needed to do this.